Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -308,11 +308,17 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
     }
 
     NSAttributedString* attributedSelf = [[[NSAttributedString alloc] initWithString:self attributes:attributes] autorelease];
+    woc::unique_cf<CTFramesetterRef> framesetter{ CTFramesetterCreateWithAttributedString((__bridge CFAttributedStringRef)attributedSelf) };
 
-    CTFramesetterRef framesetter = CTFramesetterCreateWithAttributedString((__bridge CFAttributedStringRef)attributedSelf);
-    CFAutorelease(framesetter);
+    if (size.width == 0.0) {
+        size.width = FLT_MAX;
+    }
+
+    if (size.height == 0.0) {
+        size.height = FLT_MAX;
```

Comment:
>FLT_MAX; [](start = 22, length = 8)

nit: std::numeric_limits<CGFloat>::max(), for consistency with the rest of this file?